mmcv/ops/csrc/pytorch/cpu/assign_score_withk_cpu.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright(c) OpenMMLab.All rights reserved.
+#include "pytorch_cpp_helper.hpp"
+#include "pytorch_device_registry.hpp"
+
+template <typename T>
+void assign_score_withk_forward_cpu_kernel(int B, int N0, int N1, int M, int K,
+                                           int O, int aggregate,
+                                           const T *points, const T *centers,
+                                           const T *scores,
+                                           const int64_t *knn_idx, T *output) {
+  int b, o, n, k, m;
+  for (b = 0; b < B; b++) {
+    for (n = 0; n < N1; n++) {
+      int centerIndex = knn_idx[b * N1 * O + n * O];
+      const T *c = centers + b * N0 * M * O + centerIndex * M * O;
+      for (k = 0; k < K; k++) {
+        int pointIndex = knn_idx[b * N1 * O + n * O + k];
+        const T *p = points + b * N0 * M * O + pointIndex * M * O;
+        for (m = 0; m < M; m++) {
+          for (o = 0; o < O; o++) {
+            output[b * O * N1 * K + o * N1 * K + n * K + k] +=
```

Comment:
`+=` is not necessary, `=` should be enough.

Reply:
We can't replace `+=` to `=` for every output element is the result of dot product of two vectors.